kong/plugins/jwt/handler.lua

```diff
@@ -60,10 +73,27 @@ local function retrieve_token(conf)
       end
 
       if m and #m > 0 then
-        return m[1]
+        if m[1] ~= "" then
+          token_set[m[1]] = true
+        end
       end
     end
   end
+
+  local tokens = {}
+  for token, _ in pairs(token_set) do
+    table_insert(tokens, token)
```

Comment:
If we have to keep the return value a table. Let's not use `table_insert` in here if possible. Instead, use a variable `tokens_n` to keep track of number of items and insert using tokens[tokens_n] instead.

The multiple `#tokens` checks below can also be removed this way.

Reply:
Good suggestion.